plugins/woocommerce/src/Internal/Features/FeaturesController.php

```diff
@@ -743,25 +755,18 @@ private function display_notices_in_plugins_page(): void {
 			return;
 		}
 
-		if ( 'plugins' !== get_current_screen()->id ) {
-			return;
+		$feature_filter_description_shown = $this->maybe_display_current_feature_filter_description();
+		if ( ! $feature_filter_description_shown ) {
+			$this->maybe_display_feature_incompatibility_warning();
 		}
-
-		$this->maybe_display_feature_incompatibility_warning();
-		$this->maybe_display_current_feature_filter_description();
 	}
 
 	/**
-	 * Shows a warning (on the plugins screen) when there are any incompatibility between active plugins and enabled
-	 * features.
+	 * Shows a warning when there are any incompatibility between active plugins and enabled features.
+	 * The warning is shown in on any admin screen except the plugins screen itself, since
+	 * there's already a "You are viewing
```

Comment:
It looks like some text is missing here

Reply:
Thanks, but it is not relevant any more.